lib/api/core/entrypoints/protocols/http.js

```diff
@@ -191,6 +191,9 @@ class HttpProtocol extends Protocol {
 
     // We forwarded all pipe errors to the request's event handler
     request.on('error', err => {
+      const kerr = err instanceof KuzzleError
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "kerr" [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1854)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of the unused 'kerr' variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AUnusedVariable)